cryptoki-sys/Cargo.toml

```diff
@@ -13,6 +13,7 @@ documentation = "https://docs.rs/crate/cryptoki"
 
 [build-dependencies]
 bindgen = { version = "0.57.0", optional = true }
+target-lexicon = "0.12.0"
```

Comment:
Isn't this optional?

Reply:
It is optional when `generate-bindings` is activated but I would have to create a feature that is "not `generate-bindings`" and activate it there. I don't think that's possible currently.

Or I could create a `not-generate-bindings` feature activated by default but then it gets messy. `target_lexicon` does not have any dependencies so I am fine with it.